zigup.zig

```diff
@@ -223,15 +172,17 @@ fn help() void {
         \\Uncommon Usage:
         \\
         \\  zigup fetch-index             download and print the download index json
+        \\  zigup show-config             print the current configuration in ZON format
         \\
         \\Common Options:
+        \\  --config PATH                 override the default configuration file location
```

Comment:
hmmm...not sure we'll need this new option...let's maybe hold off until we determine a new command-line option for the configuration file itself is needed.